app/test/integration_test.go

```diff
@@ -395,35 +308,20 @@ func (s *IntegrationTestSuite) TestSubmitPayForBlob_blobSizes() {
 	}
 	testCases := []testCase{
 		{
-			name:           "1,000 byte blob",
-			blob:           mustNewBlob(1_000),
-			txResponseCode: abci.CodeTypeOK,
-		},
-		{
-			name:           "10,000 byte blob",
-			blob:           mustNewBlob(10_000),
-			txResponseCode: abci.CodeTypeOK,
-		},
-		{
-			name:           "100,000 byte blob",
-			blob:           mustNewBlob(100_000),
-			txResponseCode: abci.CodeTypeOK,
-		},
-		{
-			name:           "1,000,000 byte blob",
-			blob:           mustNewBlob(1_000_000),
```

Comment:
these test cases are all redundant b/c the expected txResponseCode is identical

Reply:
it looks like the purpose of this test is test different sizes, so if we remove these then I think we should rename it or remove it entirely if the error for too large of txs is being tested elsewhere

Reply:
Good point. We're not testing for `ErrTxTooLarge` elsewhere but it seems highly related to a different test: `TestErrTotalBlobSizeTooLarge` which checks for a ErrTotalBlobSizeTooLarge error.

Since this test still tests two different blob sizes (1 mebibyte, 10,000,000 byte), if we need to keep it then I guess the test could be renamed to `TestSubmitPayForBlob_txResponseCode`

Reply:
TBH I don't think the second test case for ErrTxTooLarge is providing much value so I'm inclined to delete it and refactor the test.

Reply:
After refactoring, I realized there's already a test for `SubmitPayForBlob` in signer_test.go so deleting this entirely.